qgreenland/config/datasets.yml

```diff
@@ -670,11 +670,18 @@ The images and data are produced in a consistent way that makes the Index time-s
 
 
 - id: gem_research_stations
-  access_method: http
+  # TODO: use HTTP access method when the GEM station data is updated to have
+  # full station descriptions.
```

Comment:
Add comment to the above TODO to remind us to cleanup the citation text to remove the stuff we've already added.